libshvcoreqt/src/data/valuechange.h

```diff
@@ -123,18 +123,21 @@ SHVCOREQT_DECL_EXPORT bool compareValueX(const ValueChange::ValueX &value1, cons
 SHVCOREQT_DECL_EXPORT bool compareValueY(const ValueChange &value1, const ValueChange &value2, ValueType type);
 SHVCOREQT_DECL_EXPORT bool compareValueY(const ValueChange::ValueY &value1, const ValueChange::ValueY &value2, ValueType type);
 
-class SHVCOREQT_DECL_EXPORT ValueRecord : public std::vector<ValueChange>
+class SHVCOREQT_DECL_EXPORT ValueRecord : public QVector<ValueChange::ValueY>
```

Comment:
why QVector?

Reply:
Because QVector has nicer API for me and the performance difference is not important here, because the number of elements should be small by design.
But I don't insist on it at all, it's just a suggestion.

Reply:
I'm ok with this, I just wanted to know the reason.